lib/rps_game.rb

```diff
@@ -0,0 +1,38 @@
+class RPSGame
+
+  attr_reader :player1, :player2
+  SELECTION = ['Rock', 'Paper', 'Scissors'].freeze
+
+  def initialize(player1, player2 = "Computer")
+    @player1 = player1
+    @player2 = player2
+  end
+
+  def play(player_1_choice, player_2_choice = computer_choice)
+    raise "Invalid Input by #{@player1}" unless SELECTION.include?(player_1_choice)
+    raise "Invalid Input by #{@player2}" unless SELECTION.include?(player_2_choice)
+
+    return 'Draw' if player_1_choice == player_2_choice
+    return winner(player_1_choice, player_2_choice)
+  end
+
+  private
+
+  def winner(player_1_choice, player_2_choice)
```

Comment:
This is a really clean way of determining your winner - do you think you could keep using this approach if you added lizard and Spock options though, or might you be forced to take a different approach to keep your code clean?